src/BannerViewManager.js

```diff
@@ -0,0 +1,23 @@
+import { PropTypes } from 'react';
+import { requireNativeComponent, View } from 'react-native';
+
+
+const BANNER_HEIGHT_50 = { width: -1, height: 50 };
+const BANNER_HEIGHT_90 = { width: -1, height: 90 };
+const RECTANGLE_HEIGHT_250 = { width: -1, height: 250 };
```

Comment:
How about using flow here (removing constants entirely - that will also require propTypes to be rewritten to use `flow`) and e.g.:
```js
type bannerHeight = 50 | 90 | 250;
```